Makefile

```diff
@@ -94,11 +94,8 @@ integration: build
 
 benchmarks:
 	@echo "$@"
-	@cd benchmark/performanceTest ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/PerfTests . && sudo ../bin/PerfTests $(COMMIT) ../singleImage.csv 10
-
-benchmarks-comp:
-	@echo "$@"
-	@cd benchmark/comparisonTest ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/CompTests . && sudo ../bin/CompTests $(COMMIT) ../singleImage.csv 10
+	@cd benchmark/performanceTest ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/PerfTests .
+	@cd benchmark/comparisonTest ;  GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/CompTests .
```

Comment:
This seems to build the tests but no longer run them. Is there another workflow that runs them?

Reply:
No, there are none. Currently the user has to run the binaries themselves. Do you see value in adding a make target that runs both the benchmarks with the default values? I feel it wouldn't incentivize a user from customizing their benchmark runs

Reply:
How useful is it to run with default values and no customization? If there is value to the user there, we should make it easy, and a `make` target is our usual approach for that.

Reply:
We currently have `make test` and `make integration` which run tests. I think it would be good to keep `make benchmarks` similar. We could potentially pull the build out into `make build-benchmarks` which I think makes it pretty clear.

Reply:
It would probably be the only thing a user would use, since it's straight forward and easy to run. 
However I feel that we should also add some documentation to let users know that the benchmark runs can be customizable..

Reply:
Added ```make build-benchmarks``` and ```make benchmarks``` runs both the benchmarks with default values.